apps/web/src/services/iam/components/UserManagementStatusModal.vue

```diff
@@ -59,13 +66,46 @@ const state = reactive({
     filteredServices: undefined,
     filteredItems: [],
     filteredUniqueItems: [],
+    selectedOnlyWorkspaceUsers,
 });
 
+watch(
+    [
+        () => workspaceUserList.value,
+        () => userGroupListData.value,
+        () => state.filteredUniqueItems,
+        () => serviceListQuery.data.value,
+    ],
+    ([workspaceUsers, userGroups, filteredItems, serviceList]) => {
+        if (!workspaceUsers || !userGroups || !filteredItems?.length || !serviceList) return;
+
+        selectedOnlyWorkspaceUsers.value = workspaceUsers
```

Comment:
nitpick: We typically assign values to `state` objects using 

```typescript
state.something = ...
```

instead of 
```typescript
something.value = ...!
```